app/controllers/application_controller.rb

```diff
@@ -30,19 +29,19 @@ def options
     
 protected
   
-  def rescue_action(env)
+  def rescue_action(_env)
     respond_to do |wants|
       wants.json { render :json => "Error", :status => 500 }
     end
   end
 
-  def cors_set_access_control_headers
+  def cors_set_access_control_headers # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Maybe, 20, we will see. BTW, Sandi Metz says `Five lines per method` ( [Sandi Metz' Rules For Developers](https://robots.thoughtbot.com/sandi-metz-rules-for-developers) )